index.html

```diff
@@ -61,6 +61,164 @@ <h2 class="lets-connect">LET’S CONNECT</h2>
+          <li> <a class="year" href="#">Year</a></li>
+        </ul>
+        <p class="paragraph2">A daily selection of privately personalized reads; no accounts or sign-ups
+          required.</p>
+        <ul class="html-tags">
+          <li><a href=""><img src="./images/html.svg" alt="html-tag"></a></li>
+          <li><a href=""><img src="./images/css.svg" alt="css-tag"></a></li>
+          <li><a href=""><img src="./images/javascript.svg" alt="javascript-tag"></a></li>
+        </ul>
+        <button type="button" class="btn-card">See Project</button>
+      </section>
+    </section>
+
+    <section class="work-section">
+      <div class="image1">
+        <img src="images//snapshot4.svg" alt="stories-image">
+      </div>
+      <section class="left-block">
+        <h2 class="title"> Multi-Post
+          Stories </h2>
+        <ul class="menu2">
+          <li><a class="canopy" href="#">CANOPY</a> </li>
+          <li><img src="./images/Counter.svg" alt="counter"></li>
+          <li><a class="back-end" href="#">Back End Dev</a></li>
+          <li><img src="./images/Counter.svg" alt="counter"></li>
+          <li> <a class="year" href="#">Year</a></li>
+        </ul>
+        <p class="paragraph2">A daily selection of privately personalized reads; no accounts or sign-ups
+          required.</p>
+        <ul class="html-tags">
+          <li><a href=""><img src="./images/html.svg" alt="html-tag"></a></li>
+          <li><a href=""><img src="./images/css.svg" alt="css-tag"></a></li>
+          <li><a href=""><img src="./images/javascript.svg" alt="javascript-tag"></a></li>
+        </ul>
+        <button type="button" class="btn-card">See Project</button>
+      </section>
+    </section>
+    
+    <Section class="about-myself">
+      <div class="header-myself">
+          <h2 class="heading-about">About <br> Myself</h2>
+      </div>
+
+      <div class="paragraph3">
+          <p class="p3">Hello I’m a software developer! I can help you build a product ,
+              feature or website Look through some of my work and experience! If you like what you see and
+              have a
+              project you need coded, don’t hestiate to contact me.
+          </p>
+          <h2 class="lets-connect">LET’S CONNECT</h2>
+      </div>
+
+      <div class="social-footer">
+          <ul class="social-media">
+              <li><a href="#"><img src="./images/twitter.png" alt="twitter-icon"></a></li>
+              <li><a href="#"><img src="./images/linkedin.png" alt="linkedin-icon"></a></li>
+              <li><a href="#"><img src="./images/mail.png" alt="mail-icon"></a></li>
+              <li><a href="#"><img src="./images/github.png" alt="github-icon"></a>
+              <li>
+              <li><a href="#"><img src="./images/fingers.png" alt="fingers-icon"></a>
+              <li>
+          </ul>
+      </div>
+      <button type="button" class="button1">Get my resume</button>
+
```

Comment:
### Your button
![button](https://user-images.githubusercontent.com/93039359/185511237-863a9954-d74c-468f-96bc-7e869463bb1d.png)

### Figma Effects
![button-effects](https://user-images.githubusercontent.com/93039359/185511252-026a95ae-cc10-4e83-9c88-e1adcf72a37e.png)

- [ ] All you buttons should have hover, pressed styling shown in the Figma template :+1: to make it look professional :thinking: